### DISCUSSION TITLE
Encore app on remote VPS (no cloud infrastructure, all locally)?

### DISCUSSION THREAD

**Discussion by infiliset** [encoredev/encore#1004] [Help] [answered]
Hi everyone! I really love Encore.dev, I'd like to ask if it's possible to deploy a small app made with Encore on a remote VPS (of course with a reverse proxy like Nginx or similar)... Is it possible? Has anyone tried it?

**Comment by marcuskohlberg** [comment, marked as answer]
Hey, welcome to Encore! :)

You can probably do it using the `encore eject` command which ejects your app as a Docker image.
See docs: https://encore.dev/docs/how-to/migrate-away#ejecting-your-app-as-a-docker-image

But depending on your use case it might make more sense to just use the built-in free development cloud that Encore provides?
See docs on usage limits for Encore cloud: https://encore.dev/docs/about/usage
